pages/news.md

```diff
@@ -5,13 +5,16 @@ permalink: /news/
 nav_order: 2
 ---
 
+**April 2025:** Scikit-HEP integrates the [Scientific Python Ecosystem Coordination Core Projects][]
```

Comment:
```suggestion
**April 2025:** Scikit-HEP joins the [Scientific Python Ecosystem Coordination Core Projects][]
```

Maybe "joins" rather than "integrates".